{"name": "test_delete_test_suite_logs_error", "status": "passed", "description": "Test logger called when API deletion fails", "start": 1743501666410, "stop": 1743501666411, "uuid": "a305bfa1-b82b-4de0-8753-4ffc19f890e4", "historyId": "64808f58e18ca9f88c6f7a5b54743d27", "testCaseId": "64808f58e18ca9f88c6f7a5b54743d27", "fullName": "src.tests.ado.test_plan.test_unit_ado_test_plan.TestPlanApiWrapperExceptions#test_delete_test_suite_logs_error", "labels": [{"name": "tag", "value": "exception_handling"}, {"name": "tag", "value": "ado_test_plan"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests.ado.test_plan"}, {"name": "suite", "value": "test_unit_ado_test_plan"}, {"name": "subSuite", "value": "TestPlanApiWrapperExceptions"}, {"name": "host", "value": "fv-az2032-69"}, {"name": "thread", "value": "2321-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.ado.test_plan.test_unit_ado_test_plan"}]}